UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -97,7 +97,8 @@ public void SaveGLTFandBin(string path, string fileName)
 			binFile.Close();
 #endif
 
-			foreach (var image in _images)
+            GL.sRGBWrite = true;
+            foreach (var image in _images)
```

Comment:
nit- tabs vs spaces

Reply:
@blgrossMS: I don't know why VS keeps doing this, I thought I fixed my default spacing settings. What are the default settings I should set? Thanks for catching this!